src/ui/server/helpers/mappings/map-currencies/submitted-answer/index.ts

```diff
@@ -0,0 +1,27 @@
+import INSURANCE_FIELD_IDS from '../../../../constants/field-ids/insurance';
+import isSupportedCurrency from '../is-supported-currency';
+
+const {
+  CURRENCY: { ALTERNATIVE_CURRENCY_CODE },
+} = INSURANCE_FIELD_IDS;
+
+/**
+ * submittedAnswer
+ * if currency is supported, then should return the currency to prepopulate the radio
+ * if currency is an alternative currency, then returns ALTERNATIVE_CURRENCY_CODE for radio to be selected
+ * @param {String} currency
+ * @returns {String} currency - empty string, currencyCode for supported currency or ALTERNATIVE_CURRENCY_CODE
+ */
+const submittedAnswer = (currency?: string) => {
+  if (!currency) {
+    return '';
+  }
+
+  if (isSupportedCurrency(currency)) {
```

Comment:
Could we use the term "standard" instead of supported"?

I think it's clearer because another currency could be chosen, and it could potentially be supported?

What do you think?